src/Api/SecretsManager/Models/Request/AccessTokenCreateRequestModel.cs

```diff
@@ -34,4 +34,13 @@ public ApiKey ToApiKey(Guid serviceAccountId)
             EncryptedPayload = EncryptedPayload,
         };
     }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+    {
+        if (ExpireAt <= DateTime.UtcNow && ExpireAt != null)
```

Comment:
```suggestion
        if (ExpireAt != null && ExpireAt <= DateTime.UtcNow)
```

The conditional will short circuit if the left most check is false. So we would want our null check first.

Here are some [docs](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators#conditional-logical-and-operator-) that will explain it better.

Reply:
Thank you Thomas!